### PR TITLE
Portfolio popup window

### PR DESCRIPTION
**Pull request by GraceAbike**
### For this milestone of portfolio website,
- I have implement a popup window that includes the project details in both desktop and mobile.

image   image

## Description
For this repository, I've created html templates and styling files in order to built a responsive portfolio.

## Built With
- `HTML`
- `CSS`
- `JS`
- `Git & Github`

## Getting Started
To get a local copy up and running follow these simple example steps.
- Create a new GitHub repo.
- Create a new branch according to the GitHub flow rules.
- Add a .gitignore file.
- Set up a linter for HTML and CSS and make sure that any files that are not relevant are ignored by git.
- Add descriptive README file to your project - please use this template but remember to customize it to your project.
A .css file that includes one class with styles for your header .
- Make sure that the last file is actually ignored by git and is not present in your GitHub repository.
- Open a pull request.

### Install
To get a local copy up and running follow these simple example steps.
- Open terminal
- Clone this project by the command.

####  You should implement the following interactions:

- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears.
- In order to associate each project with the popup window details, you must refactor the project section:
- You need to use a JavaScript array to store all of the information for all projects.
- For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.
- You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.
After that, you could implement the popup window.

### CODE REVIEW

File: javascript.js [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+      css: 'css',
+      html: 'html',
+      bootstrap: 'bootstrap',
+      ruby: 'Ruby',
+      lags: 'Ruby on rails',
+    },
+    btn: 'See Project',
+    style: {
+      style1: 'section-1',
+    },
```
Comment (tufoinnkuo10) [inline javascript.js:22]: - [ ] Great job so far. please kindly remove the up/down scrolling from the mobile version
- [ ] please centralize the popup on the mobile version so it looks like that of figma
yours
Screenshot (444)_LI
figma
Screenshot (443)

File: javascript.js [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+  },
+  {
+    class: 'background-img2 background-img',
+    class2: 'card-img1',
+    subtitle: 'Multi-Post Stories',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    image: {
+      image1: './imgs/placeholder.png',
+    },
+    langlist: {
```
Comment (tufoinnkuo10) [inline javascript.js:36]: - [x] kindly centralize the popup in the desktop version as well. great job

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
## Hi @GraceAbike 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ popup design well followed

## Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: javascript.js (hunk 2 above)
Comment (GraceAbike) [reply javascript.js:36]: @tufoinnkuo10 thanks, I will make the changes

**Review by cynthiainga — APPROVED** [review]
Hi @GraceAbike, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Good commit history ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] I think it would be better if you also remove the up/down scrolling from the desktop version 🙃 😉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**